CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ include(cmake/simd.cmake)
 
 set(CMAKE_C_STANDARD 11)
```

Comment:
This feels out of place. If this is a requirement for another library, shouldn't it be defined there?

Reply:
What makes it feel out of place?

Reply:
It seems that the codebase is using C++20. If we have dependencies that use C11, we should probably have target-specific specifications instead of global ones — either by setting the requirements in the dependency build configuration, or by explicitly setting the specification for those targets:

```cpp
add_library(MyLibrary SHARED my_c_library.c)
set_target_properties(MyLibrary PROPERTIES
    C_STANDARD 11
    C_STANDARD_REQUIRED ON
)
```

This ensures consistency and prevents compatibility issues. It also improves clarity by documenting where specific standards are used, making it easier to understand the build configuration.